easyaudit/signals/model_signals.py

```diff
@@ -289,7 +289,7 @@ def crud_flow():
                             'user_id': user_id,
                             'datetime': timezone.now(),
                             'user_pk_as_string': user_pk_as_string,
-                            'metadata': get_instance_metadata(instance)
+                            'metadata': get_instance_metadata(instance, changed_fields)
```

Comment:
Does it make sense to pass even more data, like the user_id, instance, event type? Could be useful?